adoc/extensions/sycl_khr_command_graph.adoc

```diff
@@ -21,6 +32,98 @@ table below.
 
 == Usage example
 
+The example below rewrites the application from
+<<subsec:example.sycl.application>> to demonstrate the usage of this extension.
```

Comment:
This example feels quite verbose for highlighting the extension. The original example is intended as a more complex introduction to the basics of SYCL, but that feels a bit unnecessary here.

Also given that we're only showing queue recording here and that we may want a version of an example showing explicit API usage as well, it feels like having two examples of this length would be very cumbersome.